test/sql/storage/test_buffer_manager.cpp

```diff
@@ -5,8 +5,8 @@
 #include "test_helpers.hpp"
 #include "duckdb/main/client_context.hpp"
 
+// Can't use 'using namespace std' here because of conflicting names
```

Comment:
Can't we rather rename the "align" function so the conflict is removed? That seems less error-prone.

Reply:
Yea I also thought about that, but isn't `using namespace std` just bad practice in general?
It seems we don't even need it in this case, as removing it did not mess at all with the compilation (though that could be related to clang + compiler arguments, so I'm waiting on CI to see if that holds true on every platform)